sdk-extension/opentelemetry-sdk-extension-aws/README.rst

```diff
@@ -0,0 +1,51 @@
+OpenTelemetry SDK Extension for AWS X-Ray Compatibility
+=======================================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-sdk-extension-aws.svg
+   :target: https://pypi.org/project/opentelemetry-sdk-extension-aws/
+
+
+This library provides components necessary to configure the OpenTelemetry SDK
+for tracing with AWS X-Ray.
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-sdk-extension-aws
+
+
+Usage (AWS X-Ray IDs Generator)
+-------------------------------
+
+Configure the OTel SDK TracerProvider with the provided custom IDs Generator to 
+make spans compatible with the AWS X-Ray backend tracing service.
+
+.. code-block:: python
+
+    from opentelemetry.sdk.extension.aws.trace import AwsXRayIdsGenerator
+
+    trace.set_tracer_provider(
+        TracerProvider(ids_generator=AwsXRayIdsGenerator())
+    )
+
+
+Usage (AWS X-Ray Propagator)
+----------------------------
+
+Set this environment variable to have the OTel SDK use the provided AWS X-Ray 
+Propagator:
+
+::
+
+    export OTEL_PYTHON_PROPAGATORS = aws_xray
```

Comment:
I believe the API supports [OTEL_PROPAGATORS](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-api/src/opentelemetry/propagators/__init__.py#L129) without the "PYTHON", however I do believe adding the PYTHON is necessary. Maybe open up an issue in the main repo to address this?

Reply:
Great call out, thank you! I actually tested it and `OTEL_PROPAGATORS` works :) So I've updated it to use that reference!